class Error(Exception):
    pass

class SignupFailedError(Error):
    pass
